core/types/module/__init__.py

```
@@ -7,57 +7,47 @@
+    exclude_from: list = field(default=[], converter=convert2lst)
+    support_languages: list = field(default=None, converter=convert2lst)
+    desc: Union[str] = ''
+    required_admin: bool = False
+    base: bool = False
+    doc: bool = False
+    hidden: bool = False
+    load: bool = True
+    rss: bool = False
+    required_superuser: bool = False
+    required_base_superuser: bool = False
+    command_list: CommandMatches = CommandMatches.init()
```

Ruff (0.8.2)

`CommandMatches` may be undefined, or defined from star imports

(F405)